recipe/conda_build_config.yaml

```diff
@@ -4,6 +4,10 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "11.0"                 # [osx and x86_64]
 libevent:
   - "2.1.10"
-  - "2.1.12"
+lmdb:
+  - "0.9.22"
 msgpack_c:
   - 6
+
+channel_targets:
+  - conda-forge arcticdb_libevent_2_1_10
```

Comment:
You don't need to publish to a separate label, you can keep publishing to the main channel. My main concern was that the `main` branch stays up-to-date (more or less) with our migration, but what you do on non-`main` branches is less of an issue (and can still publish to the main channel/label regularly)

Reply:
Thank you for the information, I have not read the migration bot code and did not know that `main` was only the branch it considers.